app/assets/stylesheets/ltags/PodcastTag.scss

```diff
@@ -29,6 +29,7 @@
 
     a {
       color: white;
+      text-decoration: none;
```

Comment:
nitpick (not blocking) - there's no real visual affordance that these links are links. It would be nice to add in a hover/focus effect to either show the underline or darken the link color so it's a bit more consistent with other links in the app